VMAccess/vmaccess.py

```diff
@@ -324,7 +331,9 @@ def _reset_sshd_config(sshd_file_path):
                     break
 
         # Prepare cloud init config for coreos-cloudinit
-        cfg_tempfile = "/tmp/cloudinit.cfg"
+        f = tempfile.NamedTemporaryFile(delete=False)
+        f.close()
+        cfg_tempfile = f.name
```

Comment:
OK, given our conversation about the safety of tempfile.NamedTemporaryFile(), I'm okay with this code as-is.